svm30/svm30.c

```diff
@@ -88,27 +88,31 @@ static uint32_t sensirion_calc_absolute_humidity(const int32_t *temperature,
     return ((ret >> 3) * (uint32_t)(*humidity)) / 12500;
 }
 
+static void svm_set_absolute_humidity(const int32_t *temperature,
```

Comment:
I'm not too happy with the name clash potential.. the function is called set_absolute_humidity while the argument humidity is a relative humidty. I'd change the name to `svm_set_humidity`.

Not an issue you created, but you could solve it: the return vlaue is void, but we're calling i2c which could fail - we should return the result of `sgp30_set_absolute_humdity`.

Reply:
Changed the function name as proposed. Additionally, added a check for the return code of `sgp30_set_absolute_humidity`.

Reply:
That should be an int16_t instead of int32_t if I'm not mistaken

Reply:
Yes, you're right! Weird that the compiler does not throw a warning as it has to cast the 32 bit return value to 16 bit when assigning it to the `err` variable

Reply:
We have a few of these already in the code base. gcc doesn't complain about much if one is not using `-Wall -Wextra`.